docs/en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -4,56 +4,130 @@ SilverStripe CMS needs to be installed on a web server. Content authors and webs
- * Allocate at least 48MB of memory to each PHP process. (SilverStripe can be resource hungry for some intensive operations.)
- * PHP requires a suitable CSPRNG (random number generator) source for generating random tokens, password salts etc. This can be any of the following, and most operating systems will have at least one source available:
-   * PHP 7 `random_bytes()`:
-     * `CryptGenRandom` (Windows only)
-     * `arc4random_buf` (OpenBSD & NetBSD only)
-     * `getrandom(2)` (Linux only)
-     * `/dev/urandom`
-   * PHP 5 [`random_compat`](https://github.com/paragonie/random_compat) polyfill:
-     * libsodium
-     * `/dev/urandom`
-     * [`mcrypt_create_iv()`](http://php.net/manual/en/function.mcrypt-create-iv.php)
-     * CAPICOM Utilities (`CAPICOM.Utilities.1`, Windows only)
- * Required modules: ctype, dom, fileinfo, hash, intl, mbstring, session, simplexml, tokenizer, xml.
- * At least one from each group of extensions:
-     * Image library extension (gd2, imagick)
-     * DB connector library (pdo, mysqli, pgsql)
- * Recommended configuration
-     * Dev (local development for running test framework): memory_limit 512MB
-     * Production: memory_limit = 64M
-
- * See [phpinfo()](http://php.net/manual/en/function.phpinfo.php) for more information about your environment
- * One of the following databases: 
-   * MySQL 5.6+
-   * PostgreSQL 9.4+ (requires ["silverstripe/postgresql" module](http://silverstripe.org/postgresql-module))
-     * Warning: PostgreSQL has some known issues with collations when installed on Alpine, MacOS X and BSD derivatives
-     (see [PostgreSQL FAQ](https://wiki.postgresql.org/wiki/FAQ#Why_do_my_strings_sort_incorrectly.3F)).  
-     We do not support such installations, although they still may work correctly for you.  
-     As a workaround for PostgreSQL 10+ you could manually switch to ICU collations (e.g. und-x-icu).
-     There are no known workarounds for PostgreSQL <10.
-   * [SQL Server](http://silverstripe.org/microsoft-sql-server-database/),
-     [Oracle](https://github.com/smindel/silverstripe-oracle) and
-     [SQLite](http://silverstripe.org/sqlite-database/) are not commercially supported, but are under development by our open source community.
- * One of the following web server products: 
-   * Apache 2.0+ with mod_rewrite and "AllowOverride All" set
-   * IIS 7+
-   * Support for Lighttpd, IIS 6, and other web servers may work if you are familiar with configuring those products.
- * We recommend enabling content compression (for example with mod_deflate) to speed up the delivery of HTML, CSS, and JavaScript.
- * One of the following operating systems:
-   * Linux/Unix/BSD
-   * Windows
-   * Mac OS X
-
-### PHP Requirements for older SilverStripe releases
+## PHP
+
+ * PHP >=7.1
+ * PHP extensions: `ctype`, `dom`, `fileinfo`, `hash`, `intl`, `mbstring`, `session`, `simplexml`, `tokenizer`, `xml`
+ * PHP configuration: `memory_limit` with at least `48M`
```

Comment:
What is this minimum based on? I recall a time where I needed to bump the memory_limit to `256MB` to run a dev/build on a vanilla install, but maybe the memory footprint has improved (particularly in PHP 7.x)?

Reply:
It's what was mentioned as a minimum before in one place, and another place in the same doc said "64M to 512M". In the end, I don't care, it's just taking what's already there. I've seen 4.x requests with ~15M of memory usage, so I think 48M is a safe baseline to communicate.